app/javascript/tagging/components/Tagging/Tagging.js

```diff
@@ -41,9 +41,13 @@ class Tagging extends React.Component {
   findSelectedTag = (selectedTagCategory = { id: undefined }) =>
     this.props.tags.find(tag => tag.id === selectedTagCategory.id);
 
-  isMulti = selectedTagCategory =>
-    this.findSelectedTag(selectedTagCategory) &&
-    this.findSelectedTag(selectedTagCategory).singleValue === false;
+  isMulti = (selectedTagCategory) => {
+    const selectedCategory = this.findSelectedTag(selectedTagCategory);
+    if (selectedCategory && selectedCategory.singleValue) {
```

Comment:
Is there a test for when singleValue is undefined?

Reply:
yes in first test (which is already there) singleValue is not declared at all and in that singleValue is undefined.